src/index.html

```diff
@@ -20,8 +21,348 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="/"
+        class="header__logo-link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
```

Comment:
Ensure that the image path 'images/logo.png' is correct and that the image exists in the specified directory to avoid broken image links.